src/CoreBundle/private/library/classes/TCMSPageDefinitionFile.class.php

```diff
@@ -236,6 +241,15 @@ protected function LoadPageDefVars($sPagedef)
             if (isset($staticModuleList)) {
                 $this->staticModuleList = $staticModuleList;
             }
+            if (isset($allowedRights) && \strlen($allowedRights) > 0) {
+                $rightAbbreviations = \explode(',', $allowedRights);
+                foreach ($rightAbbreviations as $rightAbbreviation) {
```

Comment:
what is an abbreviation in this context?

Reply:
renamed to "rightName"